.github/PULL_REQUEST_TEMPLATE

```diff
@@ -39,14 +39,10 @@
 
 **Checklist:** :ballot_box_with_check:
 
-<!--
-  Prefix your pull request title with [WIP] if any of these are not complete yet
-
-  Tick non-applicable items as well
--->
+<!-- Leave non-applicable items unchecked -->
```

Comment:
Since this seems to be the dominant pattern, also more intuitive, and less stuff to tick.